h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -372,6 +378,10 @@ public static final class ModelMetricsMakerSchemaV3 extends SchemaV3<ModelMetric
             level = API.Level.secondary, direction = API.Direction.INOUT, gridable = true)
     public int auuc_nbins;
 
+    @API(help = "Custom AUUC thresholds (for uplift binomial classification).",
```

Comment:
These changes enable custom thresholds to calculate metrics and could be in a different PR, but it is necessary for tests. But I can move it.